internal/app/dbstart/migrate_test.go

```diff
@@ -38,3 +38,33 @@ func TestApply(t *testing.T) {
 		t.Fatalf("expectations: %v", err)
 	}
 }
+
+func TestApplyWithDescription(t *testing.T) {
```

Comment:
@jules please update this to match the new parser. The change is mostly that it must include the driver.

Description isn't supported right now but we will leave the code paths in, can you add a skip to the test that explains that we are skipping this test because descriptions are not supported.